<launch>
  <arg name="camera"      default="camera"/>

  <include file="$(find turtlebot_bringup)/launch/3dsensor.launch"/>

  <node pkg="nodelet" type="nodelet" name="tracker_nodelet" args="load hri_planner/TrackerNodelet $(arg camera)/$(arg camera)_nodelet_manager" output="screen">
	  <param name="camera_info_file" value="$(find hri_planner)/camera_calibration/kinect_rgb.yml"/>
	  <param name="map_file" value="$(find hri_planner)/maps/marker_maps/board_test.json"/>
	  <param name="filter_param_file" value="$(find hri_planner)/params/inside_out_tracker.json"/>
	  <param name="marker_size" value="0.204"/>
	  <param name="filter_mode" value="kalman"/>
	  <remap from="/inside_out_tracker/image" to="/$(arg camera)/rgb/image_raw"/>
	  <remap from="/inside_out_tracker/pose2d" to="/tracking/robot_pose2d"/>
	  <remap from="/inside_out_tracker/odom" to="/odom"/>
  </node>

  <!--node pkg="hri_planner" type="imu_manager_node" name="imu_manager" output="screen">
	<remap from="/imu_data_raw" to="/tracking/imu_data_raw"/>
  </node-->
</launch>
